### ISSUE TITLE
Tokenizer

### ISSUE THREAD

**Issue by chnafy** [reppy4620/Dialog#9]
Hello , 

when I execute the code, I get the following error: **'Tokenizer' object has no atribute 'ids_to_tokens**
, I don't know where it comes from. Thanks you

**Comment by reppy4620** [comment]
Thank you for letting me know that error and sorry for late reply.

Unfortunately I couldn't reproduce this error in my windows environment even if I created new conda environment and install packages following procedure described in README(updated in 05/19/2020).
Therefore I don't know why this error occurs.

To be sure, I write description of Tokenizer class below.
Tokenizer class as defined in tokenizer.py  is subclass of transformers.tokenization_bert_japanese.BertJapaneseTokenizer, so "ids_to_tokens" attribute is defined in BertJapaneseTokenizer or BertTokenizer which is superclass of BertJapaneseTokenizer.
I've not changed value of "ids_to_tokens" from the superclass just because I added some methods in order to decode token_ids easily.
If you met this error when decoding token_ids(e.g. torch.tensor([1, 53, 45, 3]) to sentence by using Tokenizer.decode(token_ids) and use Windows, reinstalling "transformers" package may be solution.
